### PR TITLE
Fixed issue: #805

### PR DESCRIPTION
**Pull request by vishwesh3**
Now app pings and check whether active internet is present or not before calling API if not present show notification to open browser and complete connection .

### CODE REVIEW

**Comment by vishwesh3** [comment]
@the-dagger please review it.

**Comment by vishwesh3** [comment]
@mariobehling please have a look on it.

File: android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java [hunk 1]
```diff
@@ -229,7 +232,26 @@ public void onClick(View view) {
...
         if (NetworkUtils.haveNetworkConnection(getContext())) {
-            DataDownloadManager.getInstance().downloadSession();
+            if (NetworkUtils.isActiveInternetPresent()) {
+                //Internet is working
+                DataDownloadManager.getInstance().downloadSession();
+            } else {
+                //Device is connected to WI-FI or Mobile Data but Internet is not working
+                //show snackbar
+                //will be useful if user have blocked notification for this app
+                snackbar = Snackbar.make(getView(), R.string.waiting_for_network, Snackbar.LENGTH_INDEFINITE).setAction(R.string.snackbar_refresh_action,
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java]: Better move this piece of code to a separate method as you are calling it multiple times in the code.

Comment (vishwesh3) [inline android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java]: ok

**Comment by vishwesh3** [comment]
@the-dagger changes done.

**Pull request merged**